src/main/java/com/cloudbees/jenkins/plugins/bitbucket/api/BitbucketCommit.java

```diff
@@ -28,6 +28,11 @@
  */
 public interface BitbucketCommit {
 
+    /**
```

Comment:
changes in the API that are necessary to add the trait are perfectly fine to have in this repo, e.g. this change